src/_static/tutorial/cylc-forecasting-suite/lib/python/mecator.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this file should be renamed to `mercator.py`, with its references updated too.